frontend/common/src/components/UserProfile/ProfileSections/DiversityEquityInclusionSection.tsx

```diff
@@ -68,35 +90,140 @@ const DiversityEquityInclusionSection: React.FunctionComponent<{
                   "Label preceding what groups the user identifies as part of, followed by a colon",
               })}{" "}
             </p>{" "}
-            <ul
-              data-h2-font-weight="base(700)"
-              data-h2-padding="base(0, 0, 0, x1)"
+            {indigenousCommunities && indigenousCommunities.length > 0 && (
+              <div
+                data-h2-padding="base(x1, 0, 0, 0)"
+                data-h2-display="p-tablet(flex)"
+                data-h2-align-items="base:(center)"
+                data-h2-justify-content="base:(space-between)"
+              >
+                <ul data-h2-padding="base(0, 0, 0, x1)">
+                  <li>
+                    {intl.formatMessage(
+                      getEmploymentEquityStatement("indigenous"),
+                    )}
+                  </li>
+                  <ul data-h2-padding="base(0, 0, 0, x1)">
```

Comment:
Oops, Axe doesn't like having a `ul` with a direct child of another `ul`.
![image](https://user-images.githubusercontent.com/8978655/216999676-3f2358f2-d4ab-4bed-a2b9-ef18de815cfd.png)

Reply:
Changed with [aa9a997](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5494/commits/aa9a99765ad12d81b1eba3c42a030b93bdef6017).

Reply:
:+1: